test/web-platform-tests/to-upstream/WebCryptoAPI/getRandomValues-BufferSource-subclasses.any.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const arrays = [
+  "Int8Array",
+  "Int16Array",
+  "Int32Array",
+  "BigInt64Array",
+  "Uint8Array",
+  "Uint8ClampedArray",
+  "Uint16Array",
+  "Uint32Array",
+  "BigUint64Array"
+];
+
+for (const array of arrays) {
+  const ctor = globalThis[array];
+  test(() => {
+    class Buffer extends ctor {}
+    try {
```

Comment:
This try/catch is unnecessary, and in fact would make the test harder to debug by hiding the thrown exception if it fails. Just add a comment like `// Must not throw for the test to pass`

Reply:
Yeah, coming from a Mocha background, I was a bit surprised to not have something like `expect(fn).to.not.throw(Error)`, so that's what I went with. I simply removed the try/catch in 509564b.